README.md

```diff
@@ -1,2 +1,24 @@
 # cap
 The Common Alerting Protocol (CAP) allows emergency messages to be simultaneously disseminated over a wide variety of existing and emerging public alerting systems.
+
+## Project Purpose
+This project is for providing client libraries, and examples, in one or more somewhat popular languages for container hosted services (rust, golang, js, swift, etc.).
```

Comment:
client libraries to do what? to talk to what? I assume to talk to some CAP related services, right?

Reply:
done.. thx